app/styles/_notifications.less

```diff
@@ -3,6 +3,9 @@
 // -----------------------------------------------
 
 notification-drawer-wrapper {
+  .blank-state-pf-title {
+    font-size: 14px;
```

Comment:
Better to make this relative to the font-size-base variable

Reply:
Updated to use:

```less
// app/styles/_variables.less:
@font-size-h4: ceil(@font-size-base * 1.047); // ~14px
```
works?